src/fx/computeOptions.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * computeOptions
+ * @author Deux Huit Huit
+ * 
+ * Translate xml attributes into a js object with camelcase key properties
+ * 
+ * Example:
+ * 
+ * <div data-mykey-foo-foo="cool" data-my-key-bar="true" data-my-key-baz="10" />
+ * 
+ * will translate into
+ * 
+ * {
+ *    fooFoo: 'cool',
+ *    bar: true,
+ *    baz: 10
+ * }
+ * 
+ * Usage:
+ * 
+ * App.fx.notify('computeOptions', {element: $('div'), key: 'mykey'}, function (index, options) {});
+ */
+(function ($, undefined) {
+
+	'use strict';
+
+	var computeOptions = function (key, options) {
+		var opts = {};
+		var dataAttrPattern = new RegExp('^' + options.key);
```

Comment:
I would put it into a try/catch, because it can throw if the reg-exp is not valid

Reply:
@nitriques done !